opdracht1.js

```diff
@@ -5,32 +5,48 @@ const tvTypes = inventory.map((item) => {
     return item.type;
 });
 
-console.log(tvTypes);
+//console.log(tvTypes); <-- console.log van opdracht 1
 
 
 //Opdracht 1b
 //Maak een nieuwe array en filter op originalStock - Sold > 0
 //Indien true voeg toe aan nieuwe array
-const soldOutTvs = inventory.filter((item) => {
-    return (item.originalStock - item.sold) > 0;
-});
+//voor de bonusopdracht zet het geheel in een functie
+
+function getSoldOutTvs(){
+    const soldOutTvs = inventory.filter((item) => {
+        return (item.originalStock - item.sold) <= 0;
+    });
+    //console.log(soldOutTvs); <-- console.log van opdracht 1
+
+    return soldOutTvs;
+}
+
+
 
-console.log(soldOutTvs);
 
 //Opdracht 1c: Gebruik een array-methode om alle tv's te verzamelen (de hele objecten) die over AmbiLight beschikken. Log de uitkomst in de console.
 
-const ambilightTvs = inventory.filter((item) => {
-    return item.options.ambiLight === true;
-});
+//gebruik de filter methode om alle tv's met ambilight op te halen
+//maak een nieuwe array en stop ze daarin
+//voor de bonusopdracht:  stop het geheel in een functie
+//return het resultaat
+
+function getAmbilightTvs() {
+    const ambilightTvs = inventory.filter((item) => {
+        return item.options.ambiLight === true;
+    });
```

Comment:
`item.options.amibLight` levert óók `true` op:
```suggestion
    const ambilightTvs = inventory.filter(item => item.options.ambiLight);
```